dockers/skylab/snss2-build-indices/Biotypes.tsv

```diff
@@ -0,0 +1,66 @@
+﻿#List of gene_type or transcript_type in GENCODE and Ensembl GTF file ( see: https://www.gencodegenes.org/pages/biotypes.html)

```

Comment:
Might want to mention what the two columns are in a comment and whether they have to be case-sensitive.

Reply:
Good point!